src/binary-reversal/index.js

```diff
@@ -3,6 +3,16 @@
  *
  *  * @param {string} value
  */
-function binaryReversal(value) {}
+function binaryReversal(value) {
+
+    let valueToString = Number(value).toString(2).padStart(8, 0);
+    
+    let strReverse = valueToString.split('').reverse().join('');
+    
+    let binaryToNum = parseInt(strReverse, 2).toString(10);
+
```

Comment:
Nice. These can be `const`

Reply:
Thanks Bond...noted...am improving...thanks alot